src/components/NavBar/index.vue

```diff
@@ -0,0 +1,45 @@
+<template>
```

Comment:
ไฟล์นี้ควรเป็น NavBar.vue ไปเลย ไม่ต้องใส่ Folder

Reply:
I thought that there must to be NavSearch instead of NavBar.
Then, move SearchInput to NavSearch.

```bash
├── NavSearch
   ├── SearchInput.vue
   └── index.vue
```

Please, give me a feedback for this, I want to improve my next project. 😄

Reply:
ต้องลองเองจาร ไม่มีแบบไหนผิดเลย ขึ้นอยู่กับความชอบ 🙂